### PR TITLE
Whitelist MVX

### PR DESCRIPTION
**Pull request by 0xrmvdao**

### CODE REVIEW

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 1]
```diff
@@ -63,6 +63,46 @@
...
+          "tokenManager": "0x7569B178CcF8709E6850b3b0873Be5380e04668b",
+          "tokenManagerType": "lockUnlock"
+        },
+        {
+          "symbol": "MVX",
+          "name": "Metavault Trade",
+          "axelarChainId": "linea",
+          "tokenAddress": "0x0018D96C579121a94307249d47F053E2D687b5e7",
+          "tokenManager": "0x7569B178CcF8709E6850b3b0873Be5380e04668b",
+          "tokenManagerType": "unknown"
```
Comment (milapsheth) [inline registry/mainnet/interchain/squid.tokenlist.json:95]:
```suggestion
          "tokenManagerType": "mintBurn"
```
@canhtrinh @alanrsoares looks like token manager wasn't retrieved here

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 2]
```diff
@@ -63,6 +63,46 @@
...
+        },
+        {
+          "symbol": "MVX",
+          "name": "Metavault Trade",
+          "axelarChainId": "scroll",
+          "tokenAddress": "0x0018D96C579121a94307249d47F053E2D687b5e7",
+          "tokenManager": "0x7569B178CcF8709E6850b3b0873Be5380e04668b",
+          "tokenManagerType": "lockUnlock"
```
Comment (milapsheth) [inline registry/mainnet/interchain/squid.tokenlist.json:103]:
```suggestion
          "tokenManagerType": "mintBurn"
```
remote token for a canonical interchain token has mint burn token manager type. @canhtrinh @alanrsoares can you fix the wizard here? It would also be good to more checks such as only allowing one manager to be lockUnlock type

Comment (canhtrinh) [reply registry/mainnet/interchain/squid.tokenlist.json:103]: @milapsheth yeah we fixed the scripts

File: registry/mainnet/interchain/squid.tokenlist.json (hunk 1 above)
Comment (canhtrinh) [reply registry/mainnet/interchain/squid.tokenlist.json:95]: not sure why this was recorded as unknown. @alanrsoares can you investigate?

File: package.json [hunk 3]
```diff
@@ -9,8 +9,7 @@
     "codegen:schemas": "bun run scripts/codegen-schemas.ts",
     "codegen:chains": "bun run scripts/codegen-chains.ts",
     "codegen": "bun run codegen:schemas && bun run codegen:chains",
-    "test": "vitest --run",
-    "prepare": "husky install"
```
Comment (canhtrinh) [inline package.json:13]: why was this deleted?

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 4]
```diff
@@ -63,6 +63,46 @@
...
+      "tokenId": "0x9f25cb1fa8328de2fdfcbb16ed3e0d2bb65ee3ce573a8a0d855c6589b96d8d04",
+      "deployer": "0x4126Bb708458fe7C9126DD7854255F57ac3e1d67",
+      "originalMinter": null,
+      "prettySymbol": "MVX",
+      "decimals": 18,
+      "originAxelarChainId": "polygon",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/images/tokens/mvx.svg"
```
Comment (canhtrinh) [inline registry/mainnet/interchain/squid.tokenlist.json:77]: there was a typo in our scripts (which we'll fix). in the meantime, can you change this to `https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/mvx.svg`?

File: registry/mainnet/interchain/squid.tokenlist.json (hunk 1 above)
Comment (alanrsoares) [reply registry/mainnet/interchain/squid.tokenlist.json:95]: It was fixed in the persistance layer
